.envs/.django

```diff
@@ -16,3 +16,21 @@ CELERY_FLOWER_PASSWORD=debug
 # Opensearch
 OPENSEARCH_HOST=opensearch:9200
 OPENSEARCH_INDEX_PREFIX=scoap3-backend-local
+
+# Matomo
+MATOMO_URL='//localhost:8080/'
+MATOMO_SITE_ID=1
+
+MYSQL_USER=matomo
+MYSQL_PASSWORD=matomo
+MYSQL_DATABASE=matomo
+MYSQL_ROOT_PASSWORD=matomo
+
+MATOMO_DATABASE_HOST=matomo-db
+MATOMO_DATABASE_ADAPTER=mysql
+MATOMO_DATABASE_TABLES_PREFIX=matomo_
+MATOMO_DATABASE_USERNAME=matomo
+MATOMO_DATABASE_PASSWORD=matomo
+MATOMO_DATABASE_DBNAME=matomo
+MARIADB_AUTO_UPGRADE=1
+MARIADB_INITDB_SKIP_TZINFO=1
```

Comment:
TBH, I would remove MATOMO from local env. It is not very helpful, and it just adds overhead. We can add something like `if MATOMO_URL` in the template and then add the `script` tag.